SpaceCadetPinball/pb.cpp

```diff
@@ -270,15 +270,21 @@ void pb::frame(float dtMilliSec)
 			// Retained render prevents frame skip. The next best thing - complete refresh at fixed rate. 
 			render::update(false);
 
-			// Frame time at 60 FPS = 16.(6) ms
-			auto targetTime = 1000 / 60.0f;
+			int targetFps = options::Options.TargetUps / 2;
+			targetFps = max(targetFps, 60); // at least 60
+
+			// Frame time at 60 FPS = 16.(6) ms = (16 + 17 + 17) / 3
+			auto targetTime = (frameCounter + 1) * 1000 / targetFps - frameCounter * 1000 / targetFps;
```

Comment:
targetTime should be: auto targetTime = 1000 / targetFps; 
targetFps should be: auto targetFps = options::Options.TargetUps / 2.0f;